tests/unit_tests/utils/test_core.py

```diff
@@ -84,3 +88,14 @@ def test_remove_extra_adhoc_filters(
 ) -> None:
     remove_extra_adhoc_filters(original)
     assert expected == original
+
+
+def test_parse_boolean_string():
+    true = ("y", "Y", "yes", "True", "t", "true", "True", "On", "on", "1")
+    false = ("n", "no", "f", "false", "off", "0", "Off", "No", "N", "foo")
+
+    for y in true:
+        assert parse_boolean_string(y)
+
+    for n in false:
```

Comment:
same here - let's try to avoid having two utils doing pretty much the same thing